charts/seed-monitoring/charts/core/charts/prometheus/rules/kube-etcd3.rules.yaml

```diff
@@ -65,6 +65,30 @@ groups:
       description: Etcd3 pod {{ $labels.pod }} has seen {{ $value }} proposal failures
         within the last hour.
       summary: High number of failed etcd proposals
-
+  
   - record: shoot:etcd_object_counts:sum_by_resource
-    expr: sum(etcd_object_counts) by (resource)
\ No newline at end of file
+    expr: sum(etcd_object_counts) by (resource)
+  
+  # etcd backup failure alerts
+  - alert: KubeEtcdDeltaBackupFailed
+    expr: changes(etcdbr_snapshot_latest_revision{job="kube-etcd3-backup-restore",kind="Incr"}[15m]) < 1 and ON(job,kind) changes(etcdbr_snapshot_duration_seconds_count{job="kube-etcd3-backup-restore",kind="Incr",succeeded="true"}[15m]) < 1
+    labels:
+      service: etcd
+      severity: critical
+      type: seed
+      visibility: operator
+    annotations:
+      description: No delta snapshot for the past 15 minutes.
+      summary: Etcd delta snapshot failure.
+  - alert: KubeEtcdFullBackupFailed
+    expr: changes(etcdbr_snapshot_latest_revision{job="kube-etcd3-backup-restore",kind="Full"}[1455m]) < 1 and ON(job,kind) changes(etcdbr_snapshot_duration_seconds_count{job="kube-etcd3-backup-restore",kind="Full",succeeded="true"}[1455m]) < 1
```

Comment:
```suggestion
    expr: changes(etcdbr_snapshot_latest_revision{job="kube-etcd3-backup-restore",kind="Full"}[5m]) < 1 and ON(job,kind) changes(etcdbr_snapshot_duration_seconds_count{job="kube-etcd3-backup-restore",kind="Full",succeeded="true"}[5m]) < 1
    for: 24h
```

Reply:
why is it 24h and 15 minutes? Can we just use 24 hours?

Reply:
15mins buffer is to address the corner case, say, etcd restarts during full snapshot. But when etcd restarts since there was no full snapshot taken in the last 24hrs, it will trigger full snapshot. ideally it shouldn't take 15mins for the restart - just a safe buffer.

Reply:
ok lets keep the `expr` the same but just use `for: 1455m` then it should be fine from my side.